### PR TITLE
Removing tidyverse install from ggplot lesson

### LINKED ISSUE #237: inconsistent use of tidyverse in episodes
### What is the problem?

In episode 5, Data Wrangling and Analyses with Tidyverse, it mentions not to install tidyverse as this can take a long time. However, in episode 6, Data Visualization with ggplot2, you start the lesson by installing tidyverse.

### Location of problem (optional)

https://github.com/datacarpentry/genomics-r-intro/edit/main/episodes/06-data-visualization.Rmd

### PR DESCRIPTION
**Pull request by sstevens2**
fixes #237

This change has us install and load `ggplot2` instead of the `tidyverse` which we know takes a while to install on the AWS machines in particular.  It also loads `readr` and `dplyr` for the ggplot episode.

### CODE REVIEW

**Comment by sstevens2** [comment]
Please especially review for if we need to load any other tidyverse packages for this lesson.

File: episodes/06-data-visualization.Rmd [hunk 1]
```diff
@@ -55,18 +56,27 @@ The idea of **mapping** is crucial in **ggplot**. One familiar example is to *ma
...
 
-Now, let's load the `tidyverse` package:
+Now, let's load the `ggplot2` package:
 
 ```{r load-tidyverse}
-library(tidyverse)
+library(ggplot2)
 ```
 
+We will also re-use some of the other tidyverse packages we used in the last episode, so we need to load them as well.
```
Comment (sstevens2) [inline episodes/06-data-visualization.Rmd:71]:
```suggestion
We will also use some of the other tidyverse packages we used in the last episode, so we need to load them as well.
```

File: episodes/06-data-visualization.Rmd [hunk 2]
```diff
@@ -55,18 +56,27 @@ The idea of **mapping** is crucial in **ggplot**. One familiar example is to *ma
 
 ## Installing `tidyverse`
 
-**`ggplot2`** belongs to the [**`tidyverse`** framework](https://www.tidyverse.org/). Therefore, we will start with loading the package **`tidyverse`**. If **`tidyverse`** is not already installed, then we need to install first. If it is already installed, then we can skip the following step:
+First, we need to install the `ggplot2` package.
 
 ```{r install-tidyverse, echo=TRUE, eval=FALSE}
```
Comment (sstevens2) [inline episodes/06-data-visualization.Rmd:61]:
```suggestion
```{r install-ggplot2, echo=TRUE, eval=FALSE}
```

File: episodes/06-data-visualization.Rmd [hunk 3]
```diff
@@ -55,18 +56,27 @@ The idea of **mapping** is crucial in **ggplot**. One familiar example is to *ma
...
-install.packages("tidyverse") # Installing tidyverse package, includes ggplot2 and other packages such as dplyr, readr, tidyr
+install.packages("ggplot2")
 ```
 
-Now, let's load the `tidyverse` package:
+Now, let's load the `ggplot2` package:
 
 ```{r load-tidyverse}
```
Comment (sstevens2) [inline episodes/06-data-visualization.Rmd:67]:
```suggestion
```{r load-ggplot2}
```

File: episodes/06-data-visualization.Rmd [hunk 4]
```diff
@@ -55,18 +56,27 @@ The idea of **mapping** is crucial in **ggplot**. One familiar example is to *ma
...
-library(tidyverse)
+library(ggplot2)
 ```
 
+We will also use some of the other tidyverse packages we used in the last episode, so we need to load them as well.
+
+```{r load-tidyverse}
```
Comment (sstevens2) [inline episodes/06-data-visualization.Rmd:73]:
```suggestion
```{r load-other-pkgs}
```

**Comment by naupaka** [comment]
Thanks!!

**Pull request merged**